xbmc/guilib/GUIButtonControl.cpp

```diff
@@ -312,9 +312,9 @@ void CGUIButtonControl::SetAlpha(unsigned char alpha)
   m_alpha = alpha;
 }
 
-bool CGUIButtonControl::UpdateColors()
+bool CGUIButtonControl::UpdateColors(const CGUIListItem *item)
```

Comment:
asterisk next to the type, not variable name. coding convention. here and everywhere else.